doc/source/data/dataset.rst

```diff
@@ -29,7 +29,19 @@ is already supported.
   https://docs.google.com/drawings/d/16AwJeBNR46_TsrkOmMbGaBK7u-OPsf_V8fHjU-d2PPQ/edit
 
 
-Ray Datasets simplifies general purpose parallel GPU and CPU compute in Ray; for
+Data Loading and Preprocessing for ML Training
+==============================================
+
+Ray Datasets are designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
```

Comment:
Nit: "Ray Datasets", when referring to the library, is singular; "Ray Datasets", when referring to multiple of the `Dataset` objects, is plural. I assume that we're going for the former here?
```suggestion
Ray Datasets is designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
```